src/command_functions.php

```diff
@@ -229,7 +229,10 @@ function getPathsToCheck($f_paths): ?array
             /** @var string */
             $input_path = $input_paths[$i];
 
-            if (realpath($input_path) === realpath(dirname(__DIR__) . DIRECTORY_SEPARATOR . 'psalm')
+            if (
+                realpath($input_path) === realpath(dirname(__DIR__, 3) . DIRECTORY_SEPARATOR . 'bin' . DIRECTORY_SEPARATOR . 'psalm')
```

Comment:
Does this mean `vendor/bin/psalm` is not a symlink (or whatever equivalent Windows has for symlinks) on Windows?

Reply:
Yes. On Windows Composer, instead of using symlinks, is creating batch-files, such as `psalm.bat` with the following content:

Reply:
And bash-specific files are alike with content and adjusted paths. Not symlinks.

Reply:
For future references: when installation is run on Windows or when [`bin-compat`](https://getcomposer.org/doc/06-config.md#bin-compat) flag is set to `full` Composer generates files like this: https://github.com/composer/composer/blob/c3b76a85329c60220588e861eee02c2d0150c5ce/src/Composer/Installer/BinaryInstaller.php#L187